org/postgresql/test/jdbc3/CompositeTest.java

```diff
@@ -90,7 +90,7 @@ public void testCompositeFromTable() throws SQLException {
         pstmt.setObject(1, pgo1);
         String[] ctArr = new String[1];
         ctArr[0] = "(\"{1,2}\",{},\"(1,2.2,)\")";
-        Array pgarr1 = _conn.createArrayOf("Composites.ComplexCompositeTest", ctArr);
+        Array pgarr1 = _conn.createArrayOf("\"Composites\".\"ComplexCompositeTest\"", ctArr);
```

Comment:
This previous test failed due to BC. But this is only allowed since 1202 (so it's not really an issue)